be/src/vec/functions/function_bitmap.cpp

```diff
@@ -412,7 +412,7 @@ class FunctionBitmapCount : public IFunction {
         auto data_null_map = ColumnUInt8::create(input_rows_count, 0);
         auto& null_map = data_null_map->get_data();
 
-        auto& column = block.get_by_position(arguments[0]).column;
```

Comment:
why could not use reference here?

Reply:
Because in row 418, the `column` may be assigned to a non-nullable column